### ISSUE TITLE
Empty locations for group GROUP_SEWER

### ISSUE THREAD

**Issue by illi-kun** [CleverRaven/Cataclysm-DDA#23387]
Game version:
Experimental, actual for Apr,6,2018

Operating system:
Any

Tiles or curses:
Any

Mods active:
Default set

### Expected behavior
No records about map issues in ```debug.log```.

### Actual behavior
Complains about GROUP_SEWER:

```
1:16:8.365 MAP : D:\Cataclysm-DDA\src\map.cpp:7164: Empty locations for group GROUP_SEWER at uniform submap 0,8,-1 global 5736,3164,-1
...
1:16:19.44 MAP : D:\Cataclysm-DDA\src\map.cpp:7164: Empty locations for group GROUP_SEWER at uniform submap 7,8,-1 global 5719,3176,-1
```

### Steps to reproduce the behavior
TBD

**Comment by ZhilkinSerg** [comment]
Do you know which overmap terrains are at coordinates `5719,3176,0`, `5719,3176,-1` and `5719,3176,-2`?

**Comment by illi-kun** [comment]
I found these records too late, so no savefiles and no details. Will test it a bit more.

**Comment by illi-kun** [comment]
Maybe this info can be useful:

<details>
<summary>detailed debug log with removed constant part of text</summary>

```

at uniform submap 0,8,-1 global 5724,3164,-1
at   uniform submap 1,8,-1 global 5725,3164,-1
at   uniform submap 2,0,-1 global 5726,3156,-1
at   uniform submap 2,2,-1 global 5726,3158,-1
at   3,8,-1 global 5727,3164,-1
at   uniform submap 4,10,-1 global 5728,3166,-1
at   uniform submap 6,9,-1 global 5730,3165,-1
at   uniform submap 8,3,-1 global 5732,3159,-1
at   uniform submap 10,6,-1 global 5734,3162,-1
at   uniform submap 10,7,-1 global 5734,3163,-1
at   uniform submap 0,7,-1 global 5724,3175,-1
at   uniform submap 0,10,-1 global 5724,3178,-1
at   uniform submap 1,7,-1 global 5725,3175,-1
at   uniform submap 2,1,-1 global 5726,3169,-1
at   uniform submap 4,9,-1 global 5728,3177,-1
at   uniform submap 6,8,-1 global 5730,3176,-1
at   uniform submap 7,10,-1 global 5731,3178,-1
at   uniform submap 8,2,-1 global 5732,3170,-1
at   uniform submap 10,5,-1 global 5734,3173,-1
at   uniform submap 10,6,-1 global 5734,3174,-1
at   uniform submap 0,7,-1 global 5724,3175,-1
at   uniform submap 0,10,-1 global 5724,3178,-1
at   uniform submap 1,7,-1 global 5725,3175,-1
at   uniform submap 2,1,-1 global 5726,3169,-1
at   uniform submap 4,9,-1 global 5728,3177,-1
at   uniform submap 6,8,-1 global 5730,3176,-1
at   uniform submap 7,10,-1 global 5731,3178,-1
at   uniform submap 8,2,-1 global 5732,3170,-1
at   uniform submap 10,5,-1 global 5734,3173,-1
at   uniform submap 10,6,-1 global 5734,3174,-1
at   uniform submap 0,6,-1 global 5724,3186,-1
at   uniform submap 0,9,-1 global 5724,3189,-1
at   uniform submap 1,6,-1 global 5725,3186,-1
at   uniform submap 2,0,-1 global 5726,3180,-1
at   uniform submap 4,8,-1 global 5728,3188,-1
at   uniform submap 6,7,-1 global 5730,3187,-1
at   uniform submap 7,9,-1 global 5731,3189,-1
at   uniform submap 8,1,-1 global 5732,3181,-1
at   uniform submap 10,4,-1 global 5734,3184,-1
at   uniform submap 10,5,-1 global 5734,3185,-1
at   uniform submap 1,6,-1 global 5713,3186,-1
at   uniform submap 1,9,-1 global 5713,3189,-1
at   uniform submap 2,6,-1 global 5714,3186,-1
at   uniform submap 3,0,-1 global 5715,3180,-1
at   uniform submap 5,8,-1 global 5717,3188,-1
at   uniform submap 7,7,-1 global 5719,3187,-1
at   uniform submap 8,9,-1 global 5720,3189,-1
at   uniform submap 9,1,-1 global 5721,3181,-1
at   uniform submap 0,6,-1 global 5724,3186,-1
at   uniform submap 0,9,-1 global 5724,3189,-1
at   uniform submap 1,6,-1 global 5725,3186,-1
at   uniform submap 2,0,-1 global 5726,3180,-1
at   uniform submap 4,8,-1 global 5728,3188,-1
at   uniform submap 6,7,-1 global 5730,3187,-1
at   uniform submap 7,9,-1 global 5731,3189,-1
at   uniform submap 8,1,-1 global 5732,3181,-1
at   uniform submap 10,4,-1 global 5734,3184,-1
at   uniform submap 10,5,-1 global 5734,3185,-1
at   uniform submap 0,6,-1 global 5724,3186,-1
at   uniform submap 0,9,-1 global 5724,3189,-1
at   uniform submap 1,6,-1 global 5725,3186,-1
at   uniform submap 2,0,-1 global 5726,3180,-1
at   uniform submap 4,8,-1 global 5728,3188,-1
at   uniform submap 6,7,-1 global 5730,3187,-1
at   uniform submap 7,9,-1 global 5731,3189,-1
at   uniform submap 8,1,-1 global 5732,3181,-1
at   uniform submap 10,4,-1 global 5734,3184,-1
at   uniform submap 10,5,-1 global 5734,3185,-1
at   uniform submap 0,7,-1 global 5724,3175,-1
at   uniform submap 0,10,-1 global 5724,3178,-1
at   uniform submap 1,7,-1 global 5725,3175,-1
at   uniform submap 2,1,-1 global 5726,3169,-1
at   uniform submap 4,9,-1 global 5728,3177,-1
at   uniform submap 6,8,-1 global 5730,3176,-1
at   uniform submap 7,10,-1 global 5731,3178,-1
at   uniform submap 8,2,-1 global 5732,3170,-1
at   uniform submap 10,5,-1 global 5734,3173,-1
at   uniform submap 10,6,-1 global 5734,3174,-1
at   uniform submap 1,7,-1 global 5713,3175,-1
at   uniform submap 1,10,-1 global 5713,3178,-1
at   uniform submap 2,7,-1 global 5714,3175,-1
at   uniform submap 3,1,-1 global 5715,3169,-1
at   uniform submap 5,9,-1 global 5717,3177,-1
at   uniform submap 7,8,-1 global 5719,3176,-1
at   uniform submap 8,10,-1 global 5720,3178,-1
at   uniform submap 9,2,-1 global 5721,3170,-1
at   uniform submap 1,6,-1 global 5713,3186,-1
at   uniform submap 1,9,-1 global 5713,3189,-1
at   uniform submap 2,6,-1 global 5714,3186,-1
at   uniform submap 3,0,-1 global 5715,3180,-1
at   uniform submap 5,8,-1 global 5717,3188,-1
at   uniform submap 7,7,-1 global 5719,3187,-1
at   uniform submap 8,9,-1 global 5720,3189,-1
at   uniform submap 9,1,-1 global 5721,3181,-1
at   uniform submap 0,2,-1 global 5700,3182,-1
at   uniform submap 0,5,-1 global 5700,3185,-1
at   uniform submap 2,6,-1 global 5702,3186,-1
at   uniform submap 2,9,-1 global 5702,3189,-1
at   uniform submap 3,6,-1 global 5703,3186,-1
at   uniform submap 4,0,-1 global 5704,3180,-1
at   uniform submap 6,8,-1 global 5706,3188,-1
at   uniform submap 8,7,-1 global 5708,3187,-1
at   uniform submap 9,9,-1 global 5709,3189,-1
at   uniform submap 10,1,-1 global 5710,3181,-1
at   uniform submap 0,2,-1 global 5700,3182,-1
at   uniform submap 0,5,-1 global 5700,3185,-1
at   uniform submap 2,6,-1 global 5702,3186,-1
at   uniform submap 2,9,-1 global 5702,3189,-1
at   uniform submap 3,6,-1 global 5703,3186,-1
at   uniform submap 4,0,-1 global 5704,3180,-1
at   uniform submap 6,8,-1 global 5706,3188,-1
at   uniform submap 8,7,-1 global 5708,3187,-1
at   uniform submap 9,9,-1 global 5709,3189,-1
at   uniform submap 10,1,-1 global 5710,3181,-1
at   uniform submap 0,3,-1 global 5700,3171,-1
at   uniform submap 0,6,-1 global 5700,3174,-1
at   uniform submap 2,7,-1 global 5702,3175,-1
at   uniform submap 2,10,-1 global 5702,3178,-1
at   uniform submap 3,7,-1 global 5703,3175,-1
at   uniform submap 4,1,-1 global 5704,3169,-1
at   uniform submap 6,9,-1 global 5706,3177,-1
at   uniform submap 8,8,-1 global 5708,3176,-1
at   uniform submap 9,10,-1 global 5709,3178,-1
at   uniform submap 10,2,-1 global 5710,3170,-1
at   uniform submap 0,2,-1 global 5700,3182,-1
at   uniform submap 0,5,-1 global 5700,3185,-1
at   uniform submap 2,6,-1 global 5702,3186,-1
at   uniform submap 2,9,-1 global 5702,3189,-1
at   uniform submap 3,6,-1 global 5703,3186,-1
at   uniform submap 4,0,-1 global 5704,3180,-1
at   uniform submap 6,8,-1 global 5706,3188,-1
at   uniform submap 8,7,-1 global 5708,3187,-1
at   uniform submap 9,9,-1 global 5709,3189,-1
at   uniform submap 10,1,-1 global 5710,3181,-1
at   uniform submap 0,3,-1 global 5700,3171,-1
at   uniform submap 0,6,-1 global 5700,3174,-1
at   uniform submap 2,7,-1 global 5702,3175,-1
at   uniform submap 2,10,-1 global 5702,3178,-1
at   uniform submap 3,7,-1 global 5703,3175,-1
at   uniform submap 4,1,-1 global 5704,3169,-1
at   uniform submap 6,9,-1 global 5706,3177,-1
at   uniform submap 8,8,-1 global 5708,3176,-1
at   uniform submap 9,10,-1 global 5709,3178,-1
at   uniform submap 10,2,-1 global 5710,3170,-1
at   uniform submap 1,7,-1 global 5713,3175,-1
at   uniform submap 1,10,-1 global 5713,3178,-1
at   uniform submap 2,7,-1 global 5714,3175,-1
at   uniform submap 3,1,-1 global 5715,3169,-1
at   uniform submap 5,9,-1 global 5717,3177,-1
at   uniform submap 7,8,-1 global 5719,3176,-1
at   uniform submap 8,10,-1 global 5720,3178,-1
at   uniform submap 9,2,-1 global 5721,3170,-1
at   uniform submap 1,6,-1 global 5713,3186,-1
at   uniform submap 1,9,-1 global 5713,3189,-1
at   uniform submap 2,6,-1 global 5714,3186,-1
at   uniform submap 3,0,-1 global 5715,3180,-1
at   uniform submap 5,8,-1 global 5717,3188,-1
at   uniform submap 7,7,-1 global 5719,3187,-1
at   uniform submap 8,9,-1 global 5720,3189,-1
at   uniform submap 9,1,-1 global 5721,3181,-1
at   uniform submap 1,6,-1 global 5713,3186,-1
at   uniform submap 1,9,-1 global 5713,3189,-1
at   uniform submap 2,6,-1 global 5714,3186,-1
at   uniform submap 3,0,-1 global 5715,3180,-1
at   uniform submap 5,8,-1 global 5717,3188,-1
at   uniform submap 7,7,-1 global 5719,3187,-1
at   uniform submap 8,9,-1 global 5720,3189,-1
at   uniform submap 9,1,-1 global 5721,3181,-1
at   uniform submap 1,7,-1 global 5713,3175,-1
at   uniform submap 1,10,-1 global 5713,3178,-1
at   uniform submap 2,7,-1 global 5714,3175,-1
at   uniform submap 3,1,-1 global 5715,3169,-1
at   uniform submap 5,9,-1 global 5717,3177,-1
at   uniform submap 7,8,-1 global 5719,3176,-1
at   uniform submap 8,10,-1 global 5720,3178,-1
at   uniform submap 9,2,-1 global 5721,3170,-1
at   uniform submap 0,7,-1 global 5724,3175,-1
at   uniform submap 0,10,-1 global 5724,3178,-1
at   uniform submap 1,7,-1 global 5725,3175,-1
at   uniform submap 2,1,-1 global 5726,3169,-1
at   uniform submap 4,9,-1 global 5728,3177,-1
at   uniform submap 6,8,-1 global 5730,3176,-1
at   uniform submap 7,10,-1 global 5731,3178,-1
at   uniform submap 8,2,-1 global 5732,3170,-1
at   uniform submap 10,5,-1 global 5734,3173,-1
at   uniform submap 10,6,-1 global 5734,3174,-1
at   uniform submap 0,6,-1 global 5724,3186,-1
at   uniform submap 0,9,-1 global 5724,3189,-1
at   uniform submap 1,6,-1 global 5725,3186,-1
at   uniform submap 2,0,-1 global 5726,3180,-1
at   uniform submap 4,8,-1 global 5728,3188,-1
at   uniform submap 6,7,-1 global 5730,3187,-1
at   uniform submap 7,9,-1 global 5731,3189,-1
at   uniform submap 8,1,-1 global 5732,3181,-1
at   uniform submap 10,4,-1 global 5734,3184,-1
at   uniform submap 10,5,-1 global 5734,3185,-1
at   uniform submap 0,6,-1 global 5724,3186,-1
at   uniform submap 0,9,-1 global 5724,3189,-1
at   uniform submap 1,6,-1 global 5725,3186,-1
at   uniform submap 2,0,-1 global 5726,3180,-1
at   uniform submap 4,8,-1 global 5728,3188,-1
at   uniform submap 6,7,-1 global 5730,3187,-1
at   uniform submap 7,9,-1 global 5731,3189,-1
at   uniform submap 8,1,-1 global 5732,3181,-1
at   uniform submap 10,4,-1 global 5734,3184,-1
at   uniform submap 10,5,-1 global 5734,3185,-1
at   uniform submap 0,7,-1 global 5724,3175,-1
at   uniform submap 0,10,-1 global 5724,3178,-1
at   uniform submap 1,7,-1 global 5725,3175,-1
at   uniform submap 2,1,-1 global 5726,3169,-1
at   uniform submap 4,9,-1 global 5728,3177,-1
at   uniform submap 6,8,-1 global 5730,3176,-1
at   uniform submap 7,10,-1 global 5731,3178,-1
at   uniform submap 8,2,-1 global 5732,3170,-1
at   uniform submap 10,5,-1 global 5734,3173,-1
at   uniform submap 10,6,-1 global 5734,3174,-1
at   uniform submap 0,8,-1 global 5724,3164,-1
at   uniform submap 1,8,-1 global 5725,3164,-1
at   uniform submap 2,0,-1 global 5726,3156,-1
at   uniform submap 2,2,-1 global 5726,3158,-1
at   uniform submap 4,10,-1 global 5728,3166,-1
at   uniform submap 6,9,-1 global 5730,3165,-1
at   uniform submap 8,3,-1 global 5732,3159,-1
at   uniform submap 10,6,-1 global 5734,3162,-1
at   uniform submap 10,7,-1 global 5734,3163,-1
at   uniform submap 0,8,-1 global 5736,3164,-1
at   uniform submap 1,0,-1 global 5737,3156,-1
at   uniform submap 1,2,-1 global 5737,3158,-1
at   uniform submap 3,10,-1 global 5739,3166,-1
at   uniform submap 5,9,-1 global 5741,3165,-1
at   uniform submap 7,3,-1 global 5743,3159,-1
at   uniform submap 9,6,-1 global 5745,3162,-1
at   uniform submap 9,7,-1 global 5745,3163,-1
at   uniform submap 0,8,-1 global 5736,3164,-1
at   uniform submap 1,0,-1 global 5737,3156,-1
at   uniform submap 1,2,-1 global 5737,3158,-1
at   uniform submap 3,10,-1 global 5739,3166,-1
at   uniform submap 5,9,-1 global 5741,3165,-1
at   uniform submap 7,3,-1 global 5743,3159,-1
at   uniform submap 9,6,-1 global 5745,3162,-1
at   uniform submap 9,7,-1 global 5745,3163,-1
at   uniform submap 0,8,-1 global 5724,3164,-1
at   uniform submap 1,8,-1 global 5725,3164,-1
at   uniform submap 2,0,-1 global 5726,3156,-1
at   uniform submap 2,2,-1 global 5726,3158,-1
at   uniform submap 4,10,-1 global 5728,3166,-1
at   uniform submap 6,9,-1 global 5730,3165,-1
at   uniform submap 8,3,-1 global 5732,3159,-1
at   uniform submap 10,6,-1 global 5734,3162,-1
at   uniform submap 10,7,-1 global 5734,3163,-1
at   uniform submap 0,8,-1 global 5736,3164,-1
at   uniform submap 1,0,-1 global 5737,3156,-1
at   uniform submap 1,2,-1 global 5737,3158,-1
at   uniform submap 3,10,-1 global 5739,3166,-1
at   uniform submap 5,9,-1 global 5741,3165,-1
at   uniform submap 7,3,-1 global 5743,3159,-1
at   uniform submap 9,6,-1 global 5745,3162,-1
at   uniform submap 9,7,-1 global 5745,3163,-1
at   uniform submap 0,7,-1 global 5736,3175,-1
at   uniform submap 1,1,-1 global 5737,3169,-1
at   uniform submap 3,9,-1 global 5739,3177,-1
at   uniform submap 5,8,-1 global 5741,3176,-1
at   uniform submap 6,10,-1 global 5742,3178,-1
at   uniform submap 7,2,-1 global 5743,3170,-1
at   uniform submap 9,5,-1 global 5745,3173,-1
at   uniform submap 9,6,-1 global 5745,3174,-1
at   uniform submap 10,10,-1 global 5746,3178,-1


```
</details>

-----


Save file:

[CathedralTest.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/1885615/CathedralTest.zip)

**Comment by ZhilkinSerg** [comment]
Reproduced it in this world by teleporting to subway station on Z-level -2 on new overmap.

**Comment by BevapDin** [comment]
That's not really a problem. The overmap creates spawn points for that monster group below cities. They will spawn if there is space available to place the monsters. But the underground is usually solid rock that contains no passable tile and there is no location (what the message means) to place the monsters.

They *will* spawn if there is space (e.g. in subways or basements).

One could change the overmap generation to only place that group when there is something else but solid rock.

**Comment by illi-kun** [comment]
@BevapDin do we really need these debug messages in this case?

**Comment by ZhilkinSerg** [comment]
Should be resolved in #26992
